includes/payment-methods/class-wc-stripe-upe-payment-method.php

```diff
@@ -72,7 +63,6 @@ public function __construct( $token_service ) {
 		}
 
 		$this->enabled       = in_array( static::STRIPE_ID, $enabled_upe_methods, true );
```

Comment:
```suggestion
		$this->enabled = in_array( static::STRIPE_ID, $enabled_upe_methods, true );
```

Reply:
Sorry, fixed in 181780cfbe22e7ab11cb9515ca4f857196e8c951 as well.